guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>The data used in this guide is available as an <a href="https://www.arcgis.com/home/item.html?id=c7140ae3d7ae4fd0817181461019aa75" target="_blank">item</a>. We will start by importing some libraries and downloading and extracting the data needed for the analysis in this guide. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>